### PR TITLE
Fix building of System.Security.Cryptography.Csp/Cng test projects in msbuild/VS

### PR DESCRIPTION
**Pull request by stephentoub**
Fixes https://github.com/dotnet/corefx/issues/7047
cc: @bartonjs, @svick

### CODE REVIEW

File: src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Cng/tests/System.Security.Cryptography.Cng.Tests.csproj]: Presumably this line should be deleted?

File: src/System.Security.Cryptography.Csp/tests/System.Security.Cryptography.Csp.Tests.csproj [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)'==''">Windows_Debug</Configuration>
+  </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```
Comment (bartonjs) [inline src/System.Security.Cryptography.Csp/tests/System.Security.Cryptography.Csp.Tests.csproj]: Likewise, presumably this line should be deleted?

(Also, shouldn't each of the changes be down here, after the dir.props import?)

Comment (stephentoub) [inline src/System.Security.Cryptography.Csp/tests/System.Security.Cryptography.Csp.Tests.csproj]: > Likewise, presumably this line should be deleted?

Yes, I didn't see them.  They should be removed.

> (Also, shouldn't each of the changes be down here, after the dir.props import?)

I'm just following what's done in other platform-specific .csproj tests, e.g.
https://github.com/dotnet/corefx/blob/master/src/System.Security.Cryptography.OpenSsl/tests/System.Security.Cryptography.OpenSsl.Tests.csproj#L3-L6
https://github.com/dotnet/corefx/blob/master/src/System.Security.Principal.Windows/tests/System.Security.Principal.Windows.Tests.csproj#L3-L6
etc.

Comment (bartonjs) [inline src/System.Security.Cryptography.Csp/tests/System.Security.Cryptography.Csp.Tests.csproj]: > I'm just following what's done in other platform-specific .csproj tests

That reasoning is fine by me :)

**Comment by bartonjs** [comment]
LGTM, thanks Steve.

**Comment by stephentoub** [comment]
OSX leg passed all tests but then failed due to #7096.

**Pull request merged**